templates/content-no-footer-about-en.html

```diff
@@ -0,0 +1,17 @@
+---
+{
+	"title": "Content page - Hide 'About government' section",
+	"language": "en",
+	"altLangPrefix": "content-no-footer-about",
```

Comment:
Change for the right alternate language property
```suggestion
	"altLangPage": "content-no-footer-about-fr.html",
```